client/package.json

```diff
@@ -16,6 +16,7 @@
     "formik": "^2.0.6",
     "material-ui-flat-pagination": "^4.0.0",
     "navi": "^0.13.6",
+    "notistack": "^0.9.7",
```

Comment:
do we need separate snackbar library? Material UI should have this component included.